coalib/bearlib/aspects/Smell.py

```diff
@@ -201,11 +202,15 @@ class docs:
     max_class_length = Taste[int](
         'Represents the max number of lines for a class\'s definition.',
         (999,), default=999)
-
-    # This shall probably be removed.
-    min_class_length = Taste[int](
-        'Represents the min number of lines for a class\'s definition.',
-        (1,), default=1)
+    max_constants = Taste[int](
+        'Represents the max number of constants for a class',
+        (3,), default=3)
```

Comment:
I think those tastes don't belong in the same aspect! I could write a bear that checks only the class length but not number of constants. This is multiple aspects in one!